test/e2e-secondary-network/infra/prepare_sriov.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+pushd $THIS_DIR
+
+_usage="Usage: $0 [--setup] [--cleanup]
+                  [--sriov-interface <interface name>] [--sriov-vf-count <count>] [--host-name <name>]
+
+Deploy Antrea CNI and its native secondary network prerequisites on an existing/running K8s cluster.
```

Comment:
I guess you updated this sentence to address my this comment on prepare_cluster.sh (https://github.com/antrea-io/antrea/pull/4097#discussion_r1018236173):
> Nit - removing "control node", as we deploy to a cluster, not a node?

However, probably missed my comment on this line (https://github.com/antrea-io/antrea/pull/4097#discussion_r1018242461):
> Change the description? The current is the same as that in prepare_cluster.sh.